index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Document</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+        <!-- navbar section -->
+        <section>
+            <header class="navbar">
+                <a href="#">
+                    <h4>Rabaya Rabu</h4>
+                </a>
+                <nav>
+                    <div id="bar"> 
+                        <img src="Images/Menu.png" alt="icon">
+                        <img src="Images/Menu.png" alt="icon"> 
+                        <img src="Images/Menu.png" alt="icon">  
+                    </div>
+                </nav>
+            </header>  
+        </section>
+        <!-- headline section -->
+        <section>
+            <div class="headline">
+                    <h1 id="h1">Hey I am Rabu</h1>
+                
+                    <h4 id="h4">I am a Software Developer</h4>
+    
+                    <p id="p">
+                        I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                        don’t hestiate to contact me.
+                    </p>
```

Comment:
- [ ] [OPTIONAL] Kindly consider using `class` selectors instead of `id` selectors. This is because an ID is specific to a single element, while classes can be assigned to multiple elements on a page or throughout the website.